tensorflow/python/data/experimental/ops/interleave_ops.py

```diff
@@ -235,6 +235,10 @@ def choose_from_datasets_v2(datasets, choice_dataset):
   datasets = [tf.data.Dataset.from_tensors("foo").repeat(),
               tf.data.Dataset.from_tensors("bar").repeat(),
               tf.data.Dataset.from_tensors("baz").repeat()]
+              
```

Comment:
Could you please add this to the documentation sprint spreadsheet, rather than to the file on Github? Thank you! :)

Reply:
Sure. The instructions here at meetup were not clear :)